src/style.css

```diff
@@ -1,3 +1,79 @@
+html {
+  /* stylelint-disable-next-line font-family-name-quotes */
```

Comment:
Why do you need this?

Reply:
because it threw an error when reading the font

Reply:
Consider fixing the lint error here instead of disabling the linter